in-consideration.md

```diff
@@ -0,0 +1,15 @@
+| Wassies by Wassies      | 0x1d20a51f088492a0f1c57f047a9e30c9ab5c07ea | 2021-09-22   | 12345        | 2.05               | 30.3               | 119.68              | 11104.78              | 0.41285                  | 1852                                  |
+| MoonCats                | 0xc3f733ca98e0dad0386979eb96fb1722a1a05e69 | 2021-04-22   | 19524        | 55.18              | 146.45             | 330.24              | 20540.03              | 0.23847                  | 1580                                  |
+| goblintown.wtf          | 0xbce3781ae7ca1a5e050bd9c4c77369867ebc307e | 2022-05-19   | 9999         | 47.74              | 593.03             | 1903.89             | 65664.89              | 0.54609                  | 387                                   |
+| Sappy Seals             | 0x364c828ee171616a39897688a831c2499ad972ec | 2021-08-31   | 10000        | 275.62             | 2436.77            | 4454                | 22593.13              | 1.04652                  | 4710                                  |
+| Moonbirds               | 0x23581767a106ae21c074b2276d25e5c3e136a68b | 2022-04-15   | 10000        | 2937.75            | 14179.7            | 38302.85            | 286671.05             | 5.94782                  | 1070                                  |
+| Chimpers                | 0x80336ad7a747236ef41f47ed2c7641828a480baa | 2022-05-19   | 5555         | 93.76              | 1110.84            | 3211.5              | 21359.24              | 1.11772                  | 2268                                  |
+| Finiliar                | 0x5a0121a0a21232ec0d024dab9017314509026480 | 2022-01-06   | 10000        | 10.52              | 98.57              | 320.96              | 4645.68               | 0.23623                  | 182                                   |
+| World of Women          | 0xe785e82358879f061bc3dcac6f0444462d4b5330 | 2021-07-27   | 10000        | 33.1               | 523.52             | 1610.46             | 82420.14              | 1.79215                  | 1989                                  |
+| Art Gobblers            | 0x60bb1e2aa1c9acafb4d34f71585d7e959f387769 | 2022-10-31   | 2986         | 1.2                | 99.65              | 1066.51             | 50107.24              | 0.87069                  | 922                                   |
+| Froggy Friends Official | 0x29652c2e9d3656434bc8133c69258c8d05290f41 | 2022-03-11   | 4444         | 1.42               | 24.22              | 73.58               | 1263.22               | 0.10775                  | 230                                   |
+| Doodles                 | 0x8a90cab2b38dba80c64b7734e58ee1db38b8992e | 2021-10-16   | 10000        | 2707.32            | 9148.59            | 36105.37            | 249731.11             | 5.21551                  | 22809                                 |
```

Comment:
Doodles should probably not be in paprMEME, based on the fact that they 

- already are served by several other protocols providing instant loans
- have a bid depth significantly above the other collections, suggesting lower risk

but they are included for consideration and discussion.